ansible-ipi-install/roles/installer/tasks/main.yml

```diff
@@ -43,6 +43,8 @@
   tags: extramanifests
 - include_tasks: 55_customize_filesystem.yml
   tags: customfs
+- include_tasks: 57_check_vip_status.yml
+  tags: vip_status
 - include_tasks: 59_cleanup_bootstrap.yml
   tags: cleanup
 - include_tasks: 60_deploy_ocp.yml
```

Comment:
unrelated to this PR

Reply:
removing this file as this is not related to the #424 issue.